Docker/gis-test-stand-odata/Dockerfile

```
@@ -11,8 +11,15 @@ RUN \
 
 FROM mcr.microsoft.com/dotnet/aspnet:7.0
 
+RUN apt-get update && \
+    apt-get install -y gdal-bin jq && \
+    apt-get install -y xsltproc &&\
```

можно всё в одну строчку
```suggestion
    apt-get install -y gdal-bin jq xsltproc && \
```